app/home/controllers.js

```diff
@@ -1,8 +1,10 @@
 const dateHelpers = require('../../common/helpers/date')
+const contentfulService = require('../../common/services/contentful')
 
-function dashboard(req, res) {
+async function dashboard(req, res) {
   const currentWeek = dateHelpers.getCurrentWeekAsRange()
   const today = new Date().toISOString()
+  const content = await contentfulService.fetchEntries()
```

Comment:
I am splitting hairs a little here but the fact we are using Contentful I see more as implementation details. Would this make more sense to name the service after the specific e.g. BannerService or something to that effect.  So instead of fetchEntries you have something more akin to fetchWhatsNewContent() or something like that. 

Just a thought.

Reply:
Yeah, I think this is worth considering, it's more like a `WhatsNewContentService` I reckon, and that makes things a little clearer down the road if we wanted to put in a `MaintenanceContentService` but pass it to the same banner component or page template.

Reply:
I shall make this change in another PR, as this PR is pretty bulky as it is!